src/jquery.gridster.js

```diff
@@ -3805,7 +3803,7 @@
 		for (var x = 1; x <= opts.cols; x++) {
 			var colWidth = ((x * opts.widget_base_dimensions[0]) + ((x - 1) * opts.widget_margins[0]));
 			styles += (opts.namespace + ' [data-sizex="' + x + '"] { width:' +
-			(full_width ? (this.$wrapper.width() - this.options.widget_margins[0] * 2) : colWidth > this.$wrapper.width() ? this.$wrapper.width() : colWidth ) + 'px; }\n');
+			(full_width ? (this.$wrapper.width() - this.options.widget_margins[0] * 2) : colWidth ) + 'px; }\n');
```

Comment:
Oh good call.  Never run gridster without it being full width so it didn't even cross my mind that was why it existed..

Do you have a suggested method to fixing the issue in the video I included in my issue? (#32) - Perhaps simply an option for width/height